tools/python/influx-migration/influx_migration.py

```diff
@@ -170,30 +230,53 @@ def bucket_create_rollback(host, token, bucket_name, org, skip_verify):
         client.close()
     return True
 
-def bucket_exists(host, token, bucket_name, skip_verify=False, org=None):
+def bucket_exists(host, token, bucket_name, skip_verify=False, org_name=None):
```

Comment:
Can we separate this function up as it doesn't just check if a bucket exists. Maybe we could separate the logic with the following:
- org_exists()
- bucket_exists()
- bucket_exists_in_org()

I think if we do this it will improve readability.

Reply:
Changed to use `influxdb_client.buckets_api().find_buckets()` with `org_name` if it has been provided.